### PR TITLE
Jenkins: Fix failing K8s agent builds.

### PR DESCRIPTION
**Pull request by schnatterer**
By upgrading to (almost) latest helm chart version and explicitly installing plugin dependencies.

Note: Didn't upgrade to latest helm chart version because it lead to errors on build, such as the following:
` Remote call to JNLP4-connect connection to jenkins-agent.default.svc.cluster.local/10.43.179.183:50000`

### CODE REVIEW

**Review by pmarkiewka — CHANGES_REQUESTED** [review]
typo in "...newer agent.tag then we and update"

**Comment by schnatterer** [comment]
Typo fixed

File: scripts/jenkins/init-jenkins.sh [hunk 1]
```diff
@@ -9,6 +9,8 @@ fi
 
 # When Upgrading helm chart, also upgrade additionalPlugins in jenkins/values.yaml
 # Check for "Failed to load" in the Jenkins log and add or upgrade the plugins mentioned there appropriately.
+# In addition, check if the new helm chart version uses a newer agent.tag when we and update.
```
Comment (pmarkiewka) [inline scripts/jenkins/init-jenkins.sh:12]: "...when we and update."

Comment (schnatterer) [reply scripts/jenkins/init-jenkins.sh]: I removed the `when we and update` completely, as the whole paragraph is about upgrading.

**Pull request merged**